tests/v2/validation/provisioning/rke2/ace_test.go

```diff
@@ -0,0 +1,215 @@
+
+	newUser.Password = user.Password
+
+	standardUserClient, err := client.AsUser(newUser)
+	require.NoError(r.T(), err)
+
+	r.standardUserClient = standardUserClient
+}
+
+func (r *RKE2ACETestSuite) TestProvisioningRKE2ClusterACE() {
+	nodeRoles0 := []machinepools.NodeRoles{
+		{
+			ControlPlane: true,
+			Etcd:         false,
+			Worker:       false,
+			Quantity:     3,
+		},
+		{
+			ControlPlane: false,
+			Etcd:         true,
+			Worker:       false,
+			Quantity:     1,
+		},
+		{
+			ControlPlane: false,
+			Etcd:         false,
+			Worker:       true,
+			Quantity:     1,
+		},
+	}
+
+	tests := []struct {
+		name      string
+		nodeRoles []machinepools.NodeRoles
+		client    *rancher.Client
+	}{
+		{"Multiple Control Planes - Admin", nodeRoles0, r.client},
+		{"Multiple Control Planes - Standard", nodeRoles0, r.standardUserClient},
+	}
+
+	var name string
+	for _, tt := range tests {
+		subSession := r.session.NewSession()
+		defer subSession.Cleanup()
+
+		client, err := tt.client.WithSession(subSession)
+		require.NoError(r.T(), err)
+
+		for _, providerName := range r.providers {
+			provider := CreateProvider(providerName)
+			providerName := " Node Provider: " + provider.Name.String()
+			for _, kubeVersion := range r.kubernetesVersions {
+				name = tt.name + providerName + " Kubernetes version: " + kubeVersion
+				for _, cni := range r.cnis {
+					name += " cni: " + cni
+					r.Run(name, func() {
+						cloudCredential, err := provider.CloudCredFunc(client)
+						require.NoError(r.T(), err)
+
+						clusterName := namegen.AppendRandomString(provider.Name.String())
+						generatedPoolName := fmt.Sprintf("nc-%s-pool1-", clusterName)
+						machinePoolConfig := provider.MachinePoolFunc(generatedPoolName, namespace)
+
+						machineConfigResp, err := client.Steve.SteveType(provider.MachineConfigPoolResourceSteveType).Create(machinePoolConfig)
+						require.NoError(r.T(), err)
+
+						machinePools := machinepools.RKEMachinePoolSetup(nodeRoles0, machineConfigResp)
+
+						cluster := clusters.NewK3SRKE2ClusterConfig(clusterName, namespace, cni, cloudCredential.ID, kubeVersion, machinePools)
+
+						cluster.Spec.LocalClusterAuthEndpoint = r.localClusterAuthEndpoint
+
+						clusterResp, err := clusters.CreateK3SRKE2Cluster(client, cluster)
+						require.NoError(r.T(), err)
+
+						adminClient, err := rancher.NewClient(client.RancherConfig.AdminToken, client.Session)
+						require.NoError(r.T(), err)
+						kubeProvisioningClient, err := adminClient.GetKubeAPIProvisioningClient()
+						require.NoError(r.T(), err)
+
+						result, err := kubeProvisioningClient.Clusters(namespace).Watch(context.TODO(), metav1.ListOptions{
+							FieldSelector:  "metadata.name=" + clusterName,
+							TimeoutSeconds: &defaults.WatchTimeoutSeconds,
+						})
+						require.NoError(r.T(), err)
+
+						checkFunc := clusters.IsProvisioningClusterReady
+
+						err = wait.WatchWait(result, checkFunc)
+						assert.NoError(r.T(), err)
```

Comment:
Just a small nit here, seems like we might want to use `require.NoError` cause seems like this confirmation is kind of crucial for the rest of the test steps, right?